WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```diff
@@ -726,11 +726,6 @@ protected void startCropActivity(Uri uri) {
     protected void startGravatarUpload(final String filePath) {
         if (!TextUtils.isEmpty(filePath)) {
             final File file = new File(filePath);
-            if (!mAccount.hasAccessToken()) {
```

Comment:
In fact, we are `requeringNonNull` a few lines down:

`Objects.requireNonNull(mAccountStore.getAccessToken())`